docs/intro.rst

```diff
@@ -63,28 +69,43 @@ The ``mbdump-derived.tar.bz2`` archive contains annotations, user tags and searc
 These archives include all the data required for setting up an instance of
 CritiqueBrainz.
 
-You can automatically download the archives (``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``) and
-begin the import for the MusicBrainz database::
+There are two ways in which you can import the database dump:-
 
-   $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+1. By downloading and importing the database dump the data in a single command::
 
-.. note::
+    $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+
+2. Or by manually downloading the dumps and then import it:-
+
+i. For this, you have to download the dumps ``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``
+   from http://ftp.musicbrainz.org/pub/musicbrainz/data/fullexport/.
```

Comment:
@paramsingh  I have made the changes. Regarding the direct link suggestion, it really is a better way of mentioning things in the doc but I don't know how to get a hyperlink which always link to the latest dump file. So at present I changed the previous link to the FTP link from where one can download the dumps and added a NOTE to get the latest dumps.